custom/ewsghana/static/ewsghana/ko/ews_filter.drilldown_options.js

```diff
@@ -0,0 +1,150 @@
+//probably we need clear this file
+//copy/paste from https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/static/reports/ko/report_filter.drilldown_options.js
+
+ko.bindingHandlers.select2 = {
+    init: function(el, valueAccessor, allBindingsAccessor, viewModel) {
+      ko.utils.domNodeDisposal.addDisposeCallback(el, function() {
+        $(el).select2('destroy');
+      });
+
+      var allBindings = allBindingsAccessor(),
+          select2 = ko.utils.unwrapObservable(allBindings.select2);
+
+      $(el).select2(select2);
+    },
+    update: function (el, valueAccessor, allBindingsAccessor, viewModel) {
+        var allBindings = allBindingsAccessor();
+
+        if ("value" in allBindings) {
```

Comment:
Best practice is to use `allBindings.hasOwnProperty("value")` rather than the `in` operator.